src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -38,6 +38,7 @@
                 </div>
             </template>
         </form>
+        <button @click="refreshGit" id="git-refresh-btn" class="btn mt-2" type="submit" v-if="metadata.git_supported">Refresh git</button>
```

Comment:
* The motivation for adding this button is that `fetch` can be a slow operation (assume >10 seconds), so we only do it on demand. Given that is the case we should probably indicate that an operation is in progress e.g. by at least disabling the button.
* It feels like this button should be located nearer the form fields to which it's relevant i.e. git branch and commit selectors.

Reply:
Yeah, disabling the button along with a spinner or 'Fetching..' message would be nice. Maybe the button could go to the right of the 'Git branch' select.

Reply:
..though if you do add it to the form, need to prevent default submit behaviour with `@click.prevent`